src/metrics/metrics.service.ts

```
@@ -0,0 +1,26 @@
+import { Injectable } from '@nestjs/common';
+import { CreateMetricDto } from './dto/create-metric.dto';
+import { UpdateMetricDto } from './dto/update-metric.dto';
+
+@Injectable()
+export class MetricsService {
+  create(createMetricDto: CreateMetricDto) {
```

ESLint

[error] 'createMetricDto' is defined but never used.

(@typescript-eslint/no-unused-vars)